msvc140/pch.hpp

```diff
@@ -0,0 +1,103 @@
+#pragma once
```

Comment:
Normally I don't care what you're doing in the visual studio project file directory, but what the hell is this?  Why does visual studio need to shotgun include what looks like half or more of the standard library?